File: PEMROGRAMAN 1/PRAKTIKUM MODUL 3/Bahasa C/PRAK301_2410817310009_Muhamad_Ghazi_Rakhmadi.c
#include <stdio.h>

int main() {
    int a, b, c;

    printf("Input: ");
    scanf("%d %d %d", &a, &b, &c);

    if (a < b && a < c) {
        if (b < c) {
            printf("Output: %d %d %d\n", a, b, c);
        } else {
            printf("Output: %d %d %d\n", a, c, b);
        }
    } else if (b < a && b < c) {
        if (a < c) {
            printf("Output: %d %d %d\n", b, a, c);
        } else {
            printf("Output: %d %d %d\n", b, c, a);
        }
    } else { 
        if (a < b) {
            printf("Output: %d %d %d\n", c, a, b);
        } else {
            printf("Output: %d %d %d\n", c, b, a);
        }
    }

    return 0;
}

File: PEMROGRAMAN 1/PRAKTIKUM MODUL 3/Bahasa Python/PRAK301_2410817310009_Muhamad_Ghazi_Rakhmadi.py
angka1, angka2, angka3 = map(int, input("Input: ").split())

if angka1 <= angka2 and angka1 <= angka3:
    terkecil = angka1
    if angka2 <= angka3:
        tengah = angka2
        terbesar = angka3
    else:
        tengah = angka3
        terbesar = angka2
elif angka2 <= angka1 and angka2 <= angka3:
    terkecil = angka2
    if angka1 <= angka3:
        tengah = angka1
        terbesar = angka3
    else:
        tengah = angka3
        terbesar = angka1
else:
    terkecil = angka3
    if angka1 <= angka2:
        tengah = angka1
        terbesar = angka2
    else:
        tengah = angka2
        terbesar = angka1

print(f"Output: {terkecil} {tengah} {terbesar}")